### PR TITLE
2023-05-24-sbiobertresolve_icd10cm_generalised_augmented_en

### PR DESCRIPTION
**Pull request by jsl-models**

### CODE REVIEW

File: docs/_posts/Meryem1425/2023-05-24-sbertresolve_icd10cm_augmented_en.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+| HTG-induced pancreatitis              | PROBLEM  | M79.3        | M79.3:::F10.2:::K85.3:::T46.5X:::K85.20:::K85.32:::T50.90:::A39.8:::J98.5           | drug-induced panniculitis:::drug-induced chronic pancreatitis:::drug-induced acute pancreatitis:::drug-induced pericarditis:::alcohol-induced pancreatitis:::drug induced acute pancreatitis with infected necrosis:::drug-induced dermatomyositis:::rbn - retrobulbar neuritis:::drug-induced granulomatous mediastinitis                                                                                                                                           |
```
Comment (muhammetsnts) [inline docs/_posts/Meryem1425/2023-05-24-sbertresolve_icd10cm_augmented_en.md:148]: The results should be all correct, so please change this chunk. "HTG-induced pancreatitis" can be ;

- drug-induced pancreatitis
- pancreatitis

File: docs/_posts/Meryem1425/2023-05-24-sbiobertresolve_icd10cm_generalised_augmented_en.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+
+{:.btn-box}
+[Live Demo](https://demo.johnsnowlabs.com/healthcare/ER_ICD10_CM/){:.button.button-orange}
+[Open in Colab](https://colab.research.google.com/github/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/Certification_Trainings/Healthcare/24.Improved_Entity_Resolvers_in_SparkNLP_with_sBert.ipynb){:.button.button-orange.button-orange-trans.co.button-icon}
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/clinical/models/sbiobertresolve_icd10cm_generalised_augmented_en_4.4.2_3.0_1684930238103.zip){:.button.button-orange.button-orange-trans.arr.button-icon.hidden}
+[Copy S3 URI](s3://auxdata.johnsnowlabs.com/clinical/models/sbiobertresolve_icd10cm_generalised_augmented_en_4.4.2_3.0_1684930238103.zip){:.button.button-orange.button-orange-trans.button-icon.button-copy-s3}
+
+## How to use
+
+sbiobertresolve_icd10cm_generalised resolver model must be used with sbiobert_base_cased_mli as embeddings ner_clinical as NER model. PROBLEM set in .setWhiteList().
```
Comment (muhammetsnts) [inline docs/_posts/Meryem1425/2023-05-24-sbiobertresolve_icd10cm_generalised_augmented_en.md:35]: Please change here as:
`sbiobertresolve_icd10cm_generalised` resolver model must be used with `sbiobert_base_cased_mli` as embeddings. 

sbiobertresolve_icd10cm_generalised resolver model must be used with sbiobert_base_cased_mli as embeddings ~ner_clinical as NER model. PROBLEM set in .setWhiteList().~

We shouldn't restrict the NER model, any NER model can be used and labels whitelisted.

**Review by muhammetsnts — CHANGES_REQUESTED** [review]

**Comment by muhammetsnts** [comment]
@Meryem1425 why don't we have the ground truths in brackets? the dataset should contain the concept_name_detailed column and it should be set in the model training. @Ahmetemintek we have the GTs in the dataset right?

File: docs/_posts/Meryem1425/2023-05-24-sbertresolve_icd10cm_augmented_en.md (hunk 1 above)
Comment (Meryem1425) [reply docs/_posts/Meryem1425/2023-05-24-sbertresolve_icd10cm_augmented_en.md]: Done

File: docs/_posts/Meryem1425/2023-05-24-sbiobertresolve_icd10cm_generalised_augmented_en.md (hunk 2 above)
Comment (Meryem1425) [reply docs/_posts/Meryem1425/2023-05-24-sbiobertresolve_icd10cm_generalised_augmented_en.md]: Done

File: docs/_posts/Meryem1425/2023-05-24-sbiobertresolve_icd10cm_generalised_augmented_en.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+supported: true
+annotator: SentenceEntityResolverModel
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model maps extracted medical entities to ICD10-CM codes using sbiobert_base_cased_mli Sentence Bert Embeddings. It predicts ICD codes up to 3 characters (according to ICD10 code structure the first three characters represent general type of the injury or disease).
```
[the remark this answers is not in the document]
Comment (muhammetsnts) [reply docs/_posts/Meryem1425/2023-05-24-sbiobertresolve_icd10cm_generalised_augmented_en.md:21]: model names should be in `` -> `sbiobert_base_cased_mli`

File: docs/_posts/Meryem1425/2023-05-24-sbertresolve_icd10cm_augmented_en.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+supported: true
+annotator: SentenceEntityResolverModel
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This model maps extracted medical entities to ICD10-CM codes using `sbert_jsl_medium_uncased`. Also, it has been augmented with synonyms for making it more accurate.
```
Comment (muhammetsnts) [inline docs/_posts/Meryem1425/2023-05-24-sbertresolve_icd10cm_augmented_en.md:21]: It should be ICD-10-CM everywhere.

**Review by muhammetsnts — CHANGES_REQUESTED** [review]

File: docs/_posts/Meryem1425/2023-05-24-sbertresolve_icd10cm_augmented_en.md (hunk 4 above)
Comment (Meryem1425) [reply docs/_posts/Meryem1425/2023-05-24-sbertresolve_icd10cm_augmented_en.md]: Done

File: docs/_posts/Meryem1425/2023-05-24-sbiobertresolve_icd10cm_generalised_augmented_en.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+
+{:.btn-box}
+[Live Demo](https://demo.johnsnowlabs.com/healthcare/ER_ICD10_CM/){:.button.button-orange}
+[Open in Colab](https://colab.research.google.com/github/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/Certification_Trainings/Healthcare/24.Improved_Entity_Resolvers_in_SparkNLP_with_sBert.ipynb){:.button.button-orange.button-orange-trans.co.button-icon}
+[Download](https://s3.amazonaws.com/auxdata.johnsnowlabs.com/clinical/models/sbiobertresolve_icd10cm_generalised_augmented_en_4.4.2_3.0_1684930238103.zip){:.button.button-orange.button-orange-trans.arr.button-icon.hidden}
+[Copy S3 URI](s3://auxdata.johnsnowlabs.com/clinical/models/sbiobertresolve_icd10cm_generalised_augmented_en_4.4.2_3.0_1684930238103.zip){:.button.button-orange.button-orange-trans.button-icon.button-copy-s3}
+
+## How to use
+
+`sbiobertresolve_icd10cm_generalised` resolver model must be used with `sbiobert_base_cased_mli` as embeddings.
```
[the remark this answers is not in the document]
Comment (Meryem1425) [reply docs/_posts/Meryem1425/2023-05-24-sbiobertresolve_icd10cm_generalised_augmented_en.md:35]: Done

**Review by muhammetsnts — APPROVED** [review]
good job 💯👏

**Pull request merged**